src/dicom-parse.c

```diff
@@ -1022,34 +1019,89 @@ char *dcm_parse_frame(DcmError **error,
         .big_endian = is_big_endian(),
     };
 
+    *length = desc->rows * desc->columns * desc->samples_per_pixel * (desc->bits_allocated / 8);
```

Comment:
This line need to be broken to stay under 80 cols. Perhaps:

```suggestion
    *length = desc->rows * desc->columns * 
        desc->samples_per_pixel * (desc->bits_allocated / 8);
```